projects/vc-api-implementation.md

```diff
@@ -0,0 +1,240 @@
+
+However, recent discussions in the VC API working group suggest that APIs for the configuration of exchanges may be included in the specification. An example of these discussions is recorded in this GitHub Issues comment from the 2023-08-08 teleconference: https://github.com/w3c-ccg/vc-api/issues/285#issuecomment-1670228974 
+
+#### Automated Testing 
+
+The project is tested by the two automated test suites which have been developed by the W3C CCG for testing credential issuance (https://github.com/w3c-ccg/vc-api-issuer-test-suite) and credential verification (https://github.com/w3c-ccg/vc-api-verifier-test-suite).
+
+The project is also tested by unit tests and end-to-end tests. 
+
+Integration tests are implemented based on Energy and Mobility use cases, which have historically come from Elia and BloXmove energy and mobility as use-case requirements respectively. 
+
+### Documentation 
+
+The project contains technical documentation and tutorials to facilitate understanding of the API and experiment with the implementation: https://github.com/energywebfoundation/ssi/tree/develop/apps/vc-api/docs/tutorials 
+
+Furthermore, Energy Web has documentation on VC-API and general documentation on SSI available the Energy Web gitbook: 
+
+- https://energy-web-foundation.gitbook.io/energy-web/foundational-concepts/self-sovereign-identity-introduction
+- https://energy-web-foundation.gitbook.io/energy-web/ew-dos-technology-components-2023/identity-and-access-management-iam/iam-guides/verifiable-credential-api
+
+# Alignment with the OpenWallet Foundation Mission
+_Include a statement on alignment with the [OpenWallet Foundation mission](https://openwallet-foundation.github.io/tac/governance/charter/)._
+
+The project team believes that the project is well aligned with the OpenWallet Foundation mission. 
+
+An implementation of VC API developed and hosted by the OWF would advance the adoption of interoperable and privacy enhancing digital wallets. This is because VC API is an open specification of an interface for Verifiable Credential operations, which are relevant functions for digital wallets.
+Furthermore, as an HTTP API, the project is a technology well understood by many organizations and will plausibly accelerate their efforts to develop or integrate digital wallets. 
+
+In addition, development of a VC API implementation would promote the development and proliferation of the VC API specification itself, which is an open standard related to digital wallets. 
+
+The project has been open source since its inception and aims to be inclusive, in so far as it should be usable across a wide range of domains and infrastructure requirements.
+
+# Current Code of Conduct
+_Has this project adopted a Code of Conduct? If so, please include a link to it._
+
+https://github.com/energywebfoundation/ssi/blob/develop/apps/vc-api/CODE_OF_CONDUCT.md
```

Comment:
Is the intention to continue with this CoC or move to the OWF CoC? We need to know since it will be a requirement to get this CoC approved by Linux Foundation Europe if you would like to continue with this CoC.

Reply:
The OWF CoC is the one on page 5 of this document, right? https://8112310.fs1.hubspotusercontent-na1.net/hubfs/8112310/Exhibit%20B%20--%20Linux%20Foundation%20Europe%20Policies.pdf
I found the document under "Policies" on this Linux Foundation Europe page: https://linuxfoundation.eu/en/policies

Reply:
Agreement to future adoption of LFE CoC added here: https://github.com/openwallet-foundation/project-proposals/pull/12/commits/be8b14cf092fa9af160456a55c9645faec15cd35